### PR TITLE
#29 Set a JWT expire

### LINKED ISSUE #29: Set an expiry to JWT course payload token
Currently we are not setting a Expiry for the course payload JWT token we generate to be sent to Maizey. We will be setting 5 min as the expiry of the JWT

### PR DESCRIPTION
**Pull request by mlhess**
Fixes #29 

Please note, I have not tested this code, please test.

### CODE REVIEW

File: lti_redirect/maizey.py [hunk 1]
```diff
@@ -14,9 +15,10 @@ def __init__(self, lti_launch_data) -> None:
     def get_restructured_data(self):
       course_title = self.lti_launch_data['https://purl.imsglobal.org/spec/lti/claim/context']['title']
       lis = self.lti_launch_data['https://purl.imsglobal.org/spec/lti/claim/lis']
-
+      expiration_time = datetime.datetime.utcnow() + datetime.timedelta(seconds=300)
```
Comment (zqian) [inline lti_redirect/maizey.py]: can we make the expiration time configurable, instead of hardcoded?

Comment (mlhess) [reply lti_redirect/maizey.py]: We can?  I am not sure when we would want to adjust it?

Comment (pushyamig) [reply lti_redirect/maizey.py]: If you think this is ever going to change then we could leave it as hardcoded.

**Comment by pushyamig** [comment]
Let me run this locally

**Comment by pushyamig** [comment]
@mlhess we usually create an github issue and open a PR based on that. I have created an issue for this PR

File: lti_redirect/maizey.py (hunk 1 above)
Comment (pushyamig) [reply lti_redirect/maizey.py]: the utcnow() function is deprecated. instead we have to do this

```
`from datetime import datetime, timedelta, timezone`
expiration_time = datetime.now(timezone.utc) + timedelta(seconds=300)
```

File: lti_redirect/maizey.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import datetime
```
Comment (pushyamig) [inline lti_redirect/maizey.py]:
```suggestion
from datetime import datetime, timedelta, timezone
```

File: lti_redirect/maizey.py (hunk 1 above)
Comment (pushyamig) [inline lti_redirect/maizey.py]:
```suggestion
      expiration_time = datetime.now(timezone.utc) + timedelta(seconds=300)
```

Comment (pushyamig) [reply lti_redirect/maizey.py]: @mlhess you have to accept the suggestion. I cannot accept my own suggestion.

File: lti_redirect/maizey.py (hunk 2 above)
Comment (pushyamig) [reply lti_redirect/maizey.py]: Same here accept the suggestion

**Comment by pushyamig** [comment]
This is running fine with the suggested change

**Comment by pushyamig** [comment]
You can test this code from here https://canvas-test.it.umich.edu/courses/403334/  `pu_clrt`

**Review by pushyamig — APPROVED** [review]
Thanks for the Change!

**Pull request merged**